sphinx/writers/html5.py

```diff
@@ -70,6 +70,10 @@ def __init__(self, document: nodes.document, builder: Builder) -> None:
         self._table_row_indices = [0]
         self._fieldlist_row_indices = [0]
         self.required_params_left = 0
+        # Override docutils.writers.html5_polyglot:HTMLTranslator
+        # otherwise, nodes like <inline classes="s">...</inline> will be
+        # converted to <s>...</s> by `visit_inline`.
+        self.supported_inline_tags = set()
```

Comment:
In the superclass, this is defined as a class variable. So it would be better to override this as a class variable also.